index.html

```diff
@@ -0,0 +1,146 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="./assets/style.css" type="text/css" rel="stylesheet">
+    <link href="./assets/font.css" type="text/css" rel="stylesheet">
+</head>
+
+<body>
+    <div class="header">
+        <img src="assets/logo.png" class="logo" id="logo">
+        <ul class="horizontal-list">
+            <li><a href="#howitworks"><span id="word-spacing">How it works</span></a></li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#contacts">Contacts</a></li>
+        </ul>
+    </div>
+    <div class="one">
+        <div class="head">
+            <div class="headline">
+                <h1 id="headlineSAFOR">SAFOR</h1>
+                <h1>switching your<br>
```

Comment:
Please remove all BR's, this should be working with single text inside without any inner tags.